carpics_360.js

```diff
@@ -0,0 +1,540 @@
+CarPicsSpinnerAPI = (function() {
+    this.CarPicsSpinners = function(config) {
```

Comment:
Not a function name.  This is idiomatic to browser JavaScript.  This is declaration of a class.